src/main/java/duck/operation/Ui.java

```diff
@@ -0,0 +1,156 @@
+package duck.operation;
+
+import duck.task.TaskList;
+
+import java.util.Scanner;
+
+public class Ui {
+    private static String INDENTATION = "    ";
+    private static String HORIZON = "------------------------------------------------------";
```

Comment:
I agree with @kouyk that the `HORIZON` variable could be renamed to something more appropriate and intuitive. You may also want to consider setting it to final since there is no intention of modifying the static string anymore.